www/templates/default/html/manager/Users.tpl.php

```diff
@@ -1,30 +1,45 @@
-<?php echo $context->calendar->name ?>
- <a href="<?php echo $base_manager_url . $context->calendar->shortname ?>/users/new/" 
-	class="wdn-button wdn-button-brand">+ Add a User
-</a>
+<?php
+	$crumbs = new stdClass;
+	$crumbs->crumbs = array(
+		"Events Manager" => "/manager",
+		$context->calendar->name => $context->calendar->getManageURL(),
+		"Users & Permissions" => NULL
+	);
+	echo $savvy->render($crumbs, 'BreadcrumbBar.tpl.php');
```

Comment:
It seems like a lot of this code is duplicated in many places, I wonder if there is a way to reduce it.